tools/Pandell.targets

```diff
@@ -0,0 +1,829 @@
+<!--
+
+Pandell common targets file
+
+NOTE: Copy of the corresponding file form node-repo-init v3.0.0 repository
```

Comment:
How about `Copy of https://github.com/pandell/node-repo-init/blob/v3.0.0/sync/.nuget/Pandell.targets`